frontend-cloudbuild.yaml

```diff
@@ -81,8 +81,8 @@ steps:
       ]
 
 substitutions:
-  _IMAGE_NAME: closure-service-frontend
-  _FRONTEND_BUCKET_NAME: closuretest.duckdns.org
+  _IMAGE_NAME: closure-frontend
+  _FRONTEND_BUCKET_NAME: uniclosure.me
 
 images:
   - "gcr.io/${PROJECT_ID}/${_IMAGE_NAME}:$COMMIT_SHA"
```

Comment:
Should we set the COMMIT_SHA variable before executing the script?

Reply:
No, it is a magic variable automatically set by Cloud Build. Though I believe you would have to substitute it manually if you want to manually push a build, not via git.

Reply:
I think we should add it in the docs of deployment (building and pushing manually a build) to set this variable.

Reply:
Agreed, I also added a section for the frontend